dynawo/sources/Models/CPP/ModelNetwork/DYNModelTwoWindingsTransformer.cpp

```diff
@@ -26,65 +26,66 @@
  *
  * b, g, r, x shall be specified at the side 2 voltage.
  */
-#include <cmath>
 #include <cassert>
+#include <cmath>
 #include "PARParametersSet.h"
 
-#include "DYNModelTwoWindingsTransformer.h"
+#include "DYNBusInterface.h"
 #include "DYNCommon.h"
-#include "DYNModelRatioTapChanger.h"
-#include "DYNModelPhaseTapChanger.h"
-#include "DYNModelTapChanger.h"
+#include "DYNCurrentLimitInterface.h"
+#include "DYNDerivative.h"
+#include "DYNMacrosMessage.h"
+#include "DYNMessageTimeline.h"
 #include "DYNModelBus.h"
+#include "DYNModelConstants.h"
 #include "DYNModelCurrentLimits.h"
-#include "DYNMacrosMessage.h"
-#include "DYNTrace.h"
-#include "DYNVariableForModel.h"
+#include "DYNModelNetwork.h"
+#include "DYNModelPhaseTapChanger.h"
+#include "DYNModelRatioTapChanger.h"
+#include "DYNModelTapChanger.h"
+#include "DYNModelTwoWindingsTransformer.h"
+#include "DYNModelVoltageLevel.h"
 #include "DYNParameter.h"
-#include "DYNDerivative.h"
-#include "DYNTwoWTransformerInterface.h"
-#include "DYNCurrentLimitInterface.h"
-#include "DYNStepInterface.h"
 #include "DYNPhaseTapChangerInterface.h"
 #include "DYNRatioTapChangerInterface.h"
-#include "DYNBusInterface.h"
-#include "DYNModelConstants.h"
-#include "DYNModelNetwork.h"
-#include "DYNMessageTimeline.h"
-#include "DYNModelVoltageLevel.h"
+#include "DYNStepInterface.h"
+#include "DYNTrace.h"
```

Comment:
Please do not add a great quantity of styling related modifications (even if done automatically by a tool), it makes it very difficult to find where the modification were made and break the history when we want to compare to earlier versions. Please revert the modifications in this file not related to the update of the TapChanger API.